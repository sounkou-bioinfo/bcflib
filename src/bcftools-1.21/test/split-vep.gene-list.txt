RNF13
SIX5
RPL10
