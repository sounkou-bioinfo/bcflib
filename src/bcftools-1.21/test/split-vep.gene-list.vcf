##fileformat=VCFv4.2
##contig=<ID=1,length=249250621>
##INFO=<ID=CSQ,Number=.,Type=String,Description="Consequence annotations from Ensembl VEP. Format: Allele|Consequence|IMPACT|SYMBOL|Gene|Feature_type|Feature|BIOTYPE|EXON|INTRON|HGVSc|HGVSp|cDNA_position|CDS_position|Protein_position|Amino_acids|Codons|Existing_variation|ALLELE_NUM|DISTANCE|STRAND|FLAGS|VARIANT_CLASS|SYMBOL_SOURCE|HGNC_ID|CANONICAL|MANE_SELECT|MANE_PLUS_CLINICAL|TSL|APPRIS|CCDS|ENSP|SWISSPROT|TREMBL|UNIPARC|UNIPROT_ISOFORM|GENE_PHENO|SIFT|PolyPhen|DOMAINS|miRNA|HGVS_OFFSET|AF|AFR_AF|AMR_AF|EAS_AF|EUR_AF|SAS_AF|AA_AF|EA_AF|gnomAD_AF|gnomAD_AFR_AF|gnomAD_AMR_AF|gnomAD_ASJ_AF|gnomAD_EAS_AF|gnomAD_FIN_AF|gnomAD_NFE_AF|gnomAD_OTH_AF|gnomAD_SAS_AF|MAX_AF|MAX_AF_POPS|CLIN_SIG|SOMATIC|PHENO|PUBMED|MOTIF_NAME|MOTIF_POS|HIGH_INF_POS|MOTIF_SCORE_CHANGE|TRANSCRIPTION_FACTORS|SpliceRegion|GeneSplicer|existing_InFrame_oORFs|existing_OutOfFrame_oORFs|existing_uORFs|five_prime_UTR_variant_annotation|five_prime_UTR_variant_consequence|CADD_PHRED|CADD_RAW|Ensembl_transcriptid|LRT_pred|MutationTaster_pred|Polyphen2_HDIV_pred|Polyphen2_HVAR_pred|SIFT_pred|Uniprot_acc|VEP_canonical|LoF|LoF_filter|LoF_flags|LoF_info|REVEL|SpliceAI_pred_DP_AG|SpliceAI_pred_DP_AL|SpliceAI_pred_DP_DG|SpliceAI_pred_DP_DL|SpliceAI_pred_DS_AG|SpliceAI_pred_DS_AL|SpliceAI_pred_DS_DG|SpliceAI_pred_DS_DL|SpliceAI_pred_SYMBOL">
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO
1	1000	.	C	T	.	.	CSQ=T|missense_variant|MODERATE|RNF13|ENSG00000082996|Transcript|ENST00000344229|protein_coding|3/11||ENST00000344229.7:c.92C>T|ENSP00000341361.3:p.Pro31Leu|794|92|31|P/L|cCt/cTt||1||1||SNV|HGNC|HGNC:10057||||1|P1|CCDS3146.1|ENSP00000341361|O43567.157||UPI0000134318|O43567-1|1|tolerated(0.09)|benign(0.007)|CDD:cd02123&PANTHER:PTHR22765:SF34&PANTHER:PTHR22765&Gene3D:3.50.30.30||||||||||||||||||||||||||||||||||||||27.4|4.062662|ENST00000392894&ENST00000344229&ENST00000470151&ENST00000466478&ENST00000468648&ENST00000459632&ENST00000466795&ENST00000490631|D|D&D|B&B&.&.&.&.&.&.|B&B&.&.&.&.&.&.|T&T&D&D&T&D&D&T|O43567&O43567&C9JYY4&C9JU37&C9JRV0&C9JYN7&C9J8T4&C9J383|.&YES&.&.&.&.&.&.|||||0.323|9|6|31|22|0.00|0.00|0.00|0.00|RNF13,T|splice_acceptor_variant&non_coding_transcript_variant|HIGH|ANKUB1|ENSG00000206199|Transcript|ENST00000473672|processed_transcript||1/1|ENST00000473672.1:n.487-1G>A||||||||1||-1||SNV|HGNC|HGNC:29642||||4||||||||||||||||||||||||||||||||||||||||||extended_intronic_splice_region_variant|loss/acceptor/149846117-149846118/Medium/6.247071||||||27.4|4.062662||||||||||||||9|6|31|22|0.00|0.00|0.00|0.00|RNF13
1	2000	.	G	A	.	.	CSQ=A|downstream_gene_variant|MODIFIER|DMPK|ENSG00000104936|Transcript|ENST00000291270|protein_coding||||||||||rs767172612|1|886|-1||SNV|HGNC|HGNC:2933|YES|NM_004409.5||5|A2|CCDS12674.1|ENSP00000291270|Q09013.209|E5KR06.76|UPI0000223FEB|Q09013-9|1||||||||||||||3.968e-05|0|0|0|9.843e-05|0|0|0|0.0001825|0.0001825|gnomAD_SAS|||||||||||||||||24.5|3.428627||||||||||||||-13|-38|24|-11|0.00|0.00|0.00|0.00|SIX5,A|missense_variant|MODERATE|SIX5|ENSG00000177045|Transcript|ENST00000317578|protein_coding|1/3||ENST00000317578.7:c.14C>T|ENSP00000316842.4:p.Pro5Leu|422|14|5|P/L|cCt/cTt|rs767172612|1||-1||SNV|HGNC|HGNC:10891|YES|NM_175875.5||1|P1|CCDS12673.1|ENSP00000316842|Q8N196.159||UPI0000366E2B||1|deleterious_low_confidence(0)|benign(0.093)|PANTHER:PTHR10390&PANTHER:PTHR10390:SF40&MobiDB_lite:mobidb-lite&Low_complexity_(Seg):seg|||||||||||3.968e-05|0|0|0|9.843e-05|0|0|0|0.0001825|0.0001825|gnomAD_SAS|||||||||||||||||24.5|3.428627|ENST00000643139&ENST00000317578&ENST00000622857&ENST00000560168||D|B&B&.&.|B&B&.&.|.&D&.&.|Q8N196&Q8N196&A0A087WXX3&H0YLF6|.&YES&.&.|||||0.405|-13|-38|24|-11|0.00|0.00|0.00|0.00|SIX5,A|downstream_gene_variant|MODIFIER|MEIOSIN|ENSG00000237452|Transcript|ENST00000457052|protein_coding||||||||||rs767172612|1|4290|1||SNV|HGNC|HGNC:44318|YES|NM_001310124.2||5|P1|CCDS82368.1|ENSP00000402674|C9JSJ3.79||UPI0002840CC6||||||||||||||||3.968e-05|0|0|0|9.843e-05|0|0|0|0.0001825|0.0001825|gnomAD_SAS|||||||||||||||||24.5|3.428627||||||||||||||-13|-38|24|-11|0.00|0.00|0.00|0.00|SIX5,A|upstream_gene_variant|MODIFIER|DM1-AS|ENSG00000267395|Transcript|ENST00000586251|lncRNA||||||||||rs767172612|1|2151|1||SNV|HGNC|HGNC:53125||||2||||||||||||||||||||||3.968e-05|0|0|0|9.843e-05|0|0|0|0.0001825|0.0001825|gnomAD_SAS|||||||||||||||||24.5|3.428627||||||||||||||-13|-38|24|-11|0.00|0.00|0.00|0.00|SIX5
1	3000	.	G	T	.	.	CSQ=T|splice_donor_variant&non_coding_transcript_variant|HIGH||ENSG00000285314|Transcript|ENST00000479606|lncRNA||11/21|ENST00000479606.5:n.1295+1G>T|||||||rs767191322&COSV53149291|1||1||SNV|||YES|||2||||||||||||||||||||||4.057e-06|0|0|0|0|0|8.998e-06|0|0|8.998e-06|gnomAD_NFE||0&1|1&1|||||||extended_intronic_splice_region_variant|loss/donor/20992370-20992371/Medium/6.672249||||||33|5.100843||||||||||||||46|-29|45|-1|0.00|0.00|0.42|1.00|LZTR1,T|splice_donor_variant&non_coding_transcript_variant|HIGH|LZTR1|ENSG00000099949|Transcript|ENST00000492480|retained_intron||2/5|ENST00000492480.1:n.205+1G>T|||||||rs767191322&COSV53149291|1||1||SNV|HGNC|HGNC:6742||||3||||||||1||||||||||||||4.057e-06|0|0|0|0|0|8.998e-06|0|0|8.998e-06|gnomAD_NFE||0&1|1&1|||||||extended_intronic_splice_region_variant|loss/donor/20992370-20992371/Medium/6.672249||||||33|5.100843||||||||||||||46|-29|45|-1|0.00|0.00|0.42|1.00|LZTR1
1	4000	.	C	G	.	.	CSQ=G|splice_donor_variant|HIGH|DNASE1L1|ENSG00000013563|Transcript|ENST00000014935|protein_coding||7/8|ENST00000014935.7:c.525+1G>C||||||||1||-1||SNV|HGNC|HGNC:2957||||2|P1|CCDS14747.1|ENSP00000014935|P49184.181||UPI0000129891||||||||||||||||||||||||||||||||||||extended_intronic_splice_region_variant|loss/donor/154403268-154403269/Medium/9.500494||||||24.2|3.326590|||||||||HC|||INTRON_SIZE:78||-23|24|25|1|0.00|0.06|0.00|0.99|DNASE1L1,G|missense_variant|MODERATE|RPL10|ENSG00000147403|Transcript|ENST00000451365|protein_coding|4/4||ENST00000451365.1:c.284C>G|ENSP00000406125.1:p.Thr95Ser|284|284|95|T/S|aCc/aGc||1||1|cds_start_NF|SNV|HGNC|HGNC:10298||||1|||ENSP00000406125||B8A6G2.66|UPI000173A19B||1|tolerated_low_confidence(0.2)|possibly_damaging(0.722)|Gene3D:3.90.1170.10||||||||||||||||||||||||||||||||||||||24.2|3.326590|ENST00000393638&ENST00000369809&ENST00000369807&ENST00000369808&ENST00000014935&ENST00000309585&ENST00000451865||D&D&D&D&D&D|.&.&.&.&.&.&.|.&.&.&.&.&.&.|.&.&.&.&.&.&.|P49184&P49184&P49184&P49184&P49184&P49184&A6QRJ0|.&YES&.&.&.&.&.|||||0.444|-23|24|25|1|0.00|0.06|0.00|0.99|DNASE1L1
