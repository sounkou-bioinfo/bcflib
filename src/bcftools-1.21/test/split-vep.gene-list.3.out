1:1000 ENSG00000082996 missense_variant
1:2000 ENSG00000177045 missense_variant
1:3000 ENSG00000285314 splice_donor_variant&non_coding_transcript_variant
1:3000 ENSG00000099949 splice_donor_variant&non_coding_transcript_variant
1:4000 ENSG00000147403 missense_variant
